src/ToDoList.jsx

```diff
@@ -0,0 +1,19 @@
+export default function ToDoList({ list, removeHandler }) {
+  if (list.length === 0) return <p>할 일이 없어요!</p>;
```

Comment:
```suggestion
  if (list.length === 0) { 
    return (
      <p>할 일이 없어요!</p>
    );
  }
```

예외처리를 잘 하셨어요!
다만 if 문 등을 사용하실 때 코드 가독성을 고려해서 스코프를 명시해주시면 좋아요.

Reply:
이 부분은 사람마다 코드 스타일 취향(스코프를 명시하지 않는 것이 더 깔끔하다는)에 의해 많이 갈리는 거 같아요 😊
저도 리턴문 단독으로 있는 경우 스코프를 명시하지 않는 습관이 있었는데,
확실히 말씀처럼 스코프를 명시하면 코드 가독성이 더 좋을 거 같아요! 
특히 리턴 문 안이 불가피하게 길어지는 경우, 더 크게 느껴지겠네요 ㅎㅎ 
덕분에 좋은 포인트 얻어갑니다, 감사합니다! 🙏🏻

보면서 문득 생각난 건데, **명시적**인 것과 **코드 가독성**이 관계가 꽤 깊다는 생각이 들었어요.
진짜 사소한 예시지만 ㅋㅋㅋ if문 조건의 `list.length`를 === 으로 엄격하게 비교하지 않고,
0이 falsy 값임을 이용해서 `!!list.length`로 조건을 작성할 수도 있는데,
굳이 0을 써가며 엄격하게 비교하는 것도 명시적으로 의미를 명확히 드러내고싶단 의도가 없지 않아 있었는데요 ㅎㅎㅋㅋㅋ 
`const isToDoListEmpty = !!list.length` 이렇게도 쓸 수 있겠네요 (근데 이건 너무 투머치인가 싶은 생각도 드네요 ㅋㅋㅋ) 
어떻게 생각하시는지 궁금합니다!

Reply:
말씀하신대로 코드 작성에 있어서 다양한 표현 방법들이 있기 때문에 개인의 선호도에 따라 다를 수 있어요.
저 같은 경우에는 코드를 작성할 때 코드를 읽는 사람이 내 의도를 얼마나 잘 이해할 수 있을지 의도를 드러내는 것에 신경을 많이써요.
그러다보면 네이밍에 제일 시간을 많이 쓰게되기도 하고, 자연스럽게 관심사의 분리를 해주게되는 경우가 많아요.
또한 코드를 인지할 때 별다른 인지자원을 소모하지 않도록 코드가 정리되어있고, 명시적으로 작성하려고 노력해요.
대표적으로 축약이나, 생략을 사용하지 않아요. 코드가 짧아질수는 있지만 그만큼 코드를 읽는 사람이 추측해야하고, 사고과정을 더 거쳐야해서 인지자원이 소모되죠. 그래서 명시적게 코드 가독성과도 관계가 있다고 저는 생각해요.